lib/govuk_publishing_components/presenters/step_by_step_nav_helper.rb

```diff
@@ -92,10 +92,24 @@ def get_list_element(style)
         style == "choice" ? "ul" : "ol"
       end
 
-      def link_href(active, href)
+      def link_href(active, href, parameters = {})
+        external_url = href.start_with?('http')
```

Comment:
You could extract this out to a private function. The benefit of doing that is that you won't need the conditional check below.

e.g. the first line could be: `return href if external_url?(href)`

I'd also consider modifying the check for `active` as well.

The function could then just be three simple lines:

```
return href if external_url?(href)
return  "#content" if active
update_href_with_parameters(href, parameters)
```

`update_href_with_parameters` will only be called if the other two conditions evaluate to false.

Reply:
As stated above, you don't need to pass parameters to `update_href_with_parameters` as for now they are not going to change per step_nav.